microcontroller/config/stm32f7xx_hal_conf.h

```diff
@@ -426,6 +426,12 @@
 #endif /* USE_FULL_ASSERT */
 
 
+/* ########################### Error Handling  ############################## */
+
+void _Error_Handler(char *file, int line);
+
+#define Error_Handler() _Error_Handler(__FILE__, __LINE__)
```

Comment:
Where is this used?
Is this a fix for the assertions?

Reply:
No it is not. It is just the header of error_handler.c. It must be declared because it is used nearly everywhere in the HAL. Normally the HAL expects it to be declared in main,h (which is ugly for me) so I added it here